libseccomp/src/lib.rs

```diff
@@ -1068,4 +1079,42 @@ mod tests {
             -111
         );
     }
+
+    #[test]
+    fn test_scmp_arg_cmp_from_scmpargcompare() {
+        // scmp_arg_cmp does not implement PartialEq, that's why we destruct
+        // the struct and assert_eq the individual fields.
+
+        let scmp_arg_cmp {
+            arg,
+            op,
+            datum_a,
+            datum_b,
+        } = <scmp_arg_cmp as From<ScmpArgCompare>>::from(ScmpArgCompare {
```

Comment:
Thank you for creating the unit test.
`<scmp_arg_cmp as From<ScmpArgCompare>>` is the first time I've seen.
This may be a basic question, in the above case, what does `as` expression mean?

Reply:
`<Type as Trait>`

https://doc.rust-lang.org/book/ch19-03-advanced-traits.html#fully-qualified-syntax-for-disambiguation-calling-methods-with-the-same-name

I used this syntax (instead of `scmp_arg_cmp::from(ScmpArgCompare`) to make it explicit which impl is tested.

Reply:
Thank you! That's great.